requirements.txt

```diff
@@ -1,3 +1,3 @@
-clickhouse-driver~=0.2.0
+clickhouse-driver~=0.2.4
```

Comment:
Revert please, present in 0.2.0 either: https://github.com/mymarilyn/clickhouse-driver/blob/96b7ba448c63ca2670cc9aa70d4a0e08826fb650/clickhouse_driver/client.py#L187-L189

Why do you think it is introduced in 0.2.4 only? Could you please share a link to a backlog or smth?

Reply:
My production server has v. 0.2.4 so I simply modified version to this. According to [CH Driver changelog](https://github.com/mymarilyn/clickhouse-driver/blob/master/CHANGELOG.md#005---2017-07-16) query_id support was added in 0.0.5, but I don't think there is a necessary to downgrade